src/hotspot/cpu/x86/vm_version_x86.cpp

```diff
@@ -3220,3 +3221,17 @@ intx VM_Version::allocate_prefetch_distance(bool use_watermark_prefetch) {
     }
   }
 }
+
+bool VM_Version::is_intrinsic_supported(vmIntrinsicID id) {
+  assert(id != vmIntrinsics::_none, "must be a VM intrinsic");
+  switch (id) {
+  case vmIntrinsics::_floatToFloat16:
+  case vmIntrinsics::_float16ToFloat:
+    if (!supports_f16c() && !supports_avx512vl()) return false;
```

Comment:
Please put the return on a new line - it is too easy to just see the break.

Reply:
Okay.